docs/provider/infisical.md

```diff
@@ -0,0 +1,72 @@
+![Infisical k8s Diagram](../pictures/external-secrets-operator.png)
+
+Sync secrets from [Infisical](https://www.infisical.com) to your Kubernetes cluster using External Secrets Operator.
+
+## Authentication
+In order for the operator to fetch secrets from Infisical, it needs to first authenticate with Infisical.
+
+To authenticate, you can use [Universal Auth](https://infisical.com/docs/documentation/platform/identities/universal-auth) from [Machine identities](https://infisical.com/docs/documentation/platform/identities/machine-identities).
+
+Follow the [guide here](https://infisical.com/docs/documentation/platform/identities/universal-auth) to learn how to create and obtain a pair of Client Secret and Client ID.
+
+!!! Note
+    Support for Native Kubernetes auth will be released soon, eliminating the need to supply initial token to the cluster.
```

Comment:
I don't think this makes sense here - this is not a page for infiscal future release notes.

Reply:
Haha apologies. Will remove it